developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-aws.md

```diff
@@ -5,7 +5,15 @@ slug: "run-validator-node-using-aws"
 
 # On AWS
 
-This is a step-by-step guide to install an Aptos node on AWS. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+This is a step-by-step guide to install an Aptos node on Amazon Web Services (AWS). Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+
+:::caution Did you set up your AWS account?
```

Comment:
This seems redundant with the prerequisites below. Are folks actually forgetting to create accounts?